egs/mgb2/ASR/conformer_ctc/.nfs00000001d6f664e40000f3eb

```diff
@@ -0,0 +1,75 @@
+## Introduction
```

Comment:
Could you remove this backup file?

You can add `.nfs*` to `.gitignore` so that git won't track such files.

Reply:
> @AmirHussein96
> 
> Could you also upload the pre-trained model, decoding results, and training logs to hugging face and note down the link in `RESULTS.md`?
> 
> Others can download your pre-trained models and use them directly.
> 
> We have done this for librispeech. Please see https://github.com/k2-fsa/icefall/blob/master/egs/librispeech/ASR/RESULTS.md
> 
> <img alt="Screen Shot 2022-09-08 at 9 14 18 AM" width="1047" src="https://user-images.githubusercontent.com/5284924/189011937-37a59118-0b26-4ad0-a7ca-34076054df63.png">

I am planning to upload the pretrained models but I am not sure if I can do this right away. Let me consult with the side that supports me with the hardware resources.

Reply:
I added the ctc conformer model https://huggingface.co/AmirHussein/icefall-asr-mgb2-conformer_ctc-2022-27-06